src/proxy.rs

```diff
@@ -120,19 +145,27 @@ where
                 info!(target: "proxy::call", message = "proxying request to rollup-boost server", ?method);
                 service.inner.call(req).await.map_err(|e| e.into())
             } else if FORWARD_REQUESTS.contains(&method.as_str()) {
-                // If the request should be forwarded, send to both the
-                // default execution client and the builder
-                let builder_req =
-                    HttpRequest::from_parts(parts.clone(), HttpBody::from(body_bytes.clone()));
-                let builder_method = method.clone();
-                let mut builder_client = service.builder_client.clone();
-                tokio::spawn(async move {
-                    let _ = builder_client.forward(builder_req, builder_method).await;
-                });
-
-                let l2_req = HttpRequest::from_parts(parts, HttpBody::from(body_bytes));
-                info!(target: "proxy::call", message = "forward request to default execution client", ?method);
-                service.l2_client.forward(l2_req, method).await
+                if method == MINER_SET_MAX_DA_SIZE {
```

Comment:
would be useful here to add docs why `MINER_SET_MAX_DA_SIZE` is treated differently here

Reply:
Done!